shared/packages/worker/src/worker/accessorHandlers/lib/CachedQuantelGateway.ts

```diff
@@ -25,44 +22,24 @@ export class CachedQuantelGateway extends QuantelGateway {
 	) {
 		super(config)
 		this.cacheExpire = config?.cacheExpire ?? DEFAULT_CACHE_EXPIRE
-		setInterval(() => this.purgeCache(), PURGE_CACHE_INTERVAL)
 	}
-	async connectToISA(ISAUrls: string | string[]): Promise<ConnectionDetails> {
-		const result = this.queryCache<ConnectionDetails>('connectToISA', [ISAUrls])
-		if (result !== undefined) return result
-
-		return this.ensureInCache('connectToISA', [ISAUrls], super.connectToISA(ISAUrls))
-	}
-	async getClip(clipId: number): Promise<ClipData | null> {
-		const result = this.queryCache<ClipData>('getClip', [clipId])
-		if (result !== undefined) return result
-
-		return this.ensureInCache('getClip', [clipId], super.getClip(clipId))
+	public purgeCache(): void {
+		this._cache.clear()
 	}
+	// searchClip() is used often. By caching it, we reduce the load on the server.
 	async searchClip(searchQuery: ClipSearchQuery): Promise<ClipDataSummary[]> {
 		const result = this.queryCache<ClipDataSummary[]>('searchClip', [searchQuery])
 		if (result !== undefined) return result
 
 		return this.ensureInCache('searchClip', [searchQuery], super.searchClip(searchQuery))
 	}
-	async getZones(): Promise<ZoneInfo[]> {
```

Comment:
`getZones()` is used in `doFileCopyExpectation` and is very unlikely to change in a 3s window.